development/constants.py

```diff
@@ -1,3 +1,14 @@
 LOSS = 0
 TIE = 1
 WIN = 2
+
+STATE = 'state'
+VALID_STATE = 'valid'
+INVALID_STATE = 'invalid'
+ALL_STATES = 'All'
```

Comment:
why this constant has a Capital letter? could be a lower case instead?

Reply:
That is because it's the option that the user is gonna see inside the dropdown. But I think a better way to do this it's leaving all constants in lower case and in the Django template use the tittle method to put it in a more user way.
Thank you. I'll refactor this